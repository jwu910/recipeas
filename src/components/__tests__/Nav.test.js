import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from '../Nav';

describe ('Navigation bar', () => {
  test('renders without crashing', () => {
    const component = renderer.create(<Navigation />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
