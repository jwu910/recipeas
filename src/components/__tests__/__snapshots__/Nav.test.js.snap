// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar renders without crashing 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-top"
>
  <div
    className="container-fluid"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      Recipeas
    </a>
    <ul
      className="nav navbar-nav"
    >
      <li>
        <a
          href="https://github.com/phuchle/recipeas"
        >
          Source Code
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
